tests/jobs/test_local_task_job.py

```diff
@@ -43,6 +43,9 @@
 class TestLocalTaskJob(unittest.TestCase):
     def setUp(self):
         clear_db_runs()
+        patcher = patch('airflow.jobs.base_job.sleep')
```

Comment:
What about scheduler_job and backill_job -- shouldn't we need to patch those too?

Reply:
I guess not? - it doesn't look like the tests take any longer without it (it's within the normal variance of tests I think)

Reply:
yeah, this test doesn't cover the code path that does the actual sleep call in those two classes. we should definitely patch them if a test runs through those code path going forward.